### DISCUSSION TITLE
Translate Cookies Pop-Up

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#3173] [Questions & Answers] [answered]
Good morning everyone
I have a site multistore in multi linguage
I'm trying to translate the standard openmage pop-up text but I'm having some difficulty.

If I remember correctly, in the past I modified the text by simply enabling the "online translation" function found in:
System > Configuration > Advanced > Developer tab "Translate online" Enable in frontEnd

Now, however, this function is not enabled for that specific text...

Maybe I remember wrong.

I've also searched the local/folder/*.cvs files, but can't find it

**Comment by addison74** [comment]
My recommendation is to test different versions of Magento and OpenMage installed from scratch to determine if this feature exists and when we lost it. I will check when I have some time.

**Comment by elidrissidev** [comment, marked as answer]
Since cookie pop-up is a static block, you can't translate it that way. Instead, go to CMS -> Static Blocks and search for a block by identifier "cookie_restriction_notice_block" which should contain the content in English and set the store to English, then create other blocks with the same identifier but with a different store and change the content accordingly.

<img width="923" alt="Screen Shot 2023-04-14 at 10 30 38 AM" src="https://user-images.githubusercontent.com/67818913/232021568-e50696e6-a0d2-467a-b5ae-ae37fbf40120.png">

**Comment by Danieleeffe1** [comment]
Let's see if I understand:

If I create a block in
CMS > STATIC BLOCKS

With the id you provided (cookie_restriction_notice_block) does this override OpenMage's default block?

**Reply by elidrissidev** [reply]
No it does not, the same identifier can be used in multiple blocks, you just have to set different stores from select menu and change the content.
